widgets/instanceInfo.go

```diff
@@ -17,34 +17,33 @@ import (
 )
 
 func InstanceInfo() *text.Text {
-	appVersion, err := src.Exec_cmd(config.HarmonyPath + "./harmony -version")
-	if err!=nil {
-		appVersion = "Error collecting data\n"
-	}
-	appVersion = "App version: " + appVersion
-
-	// TODO: stop using wallet.sh to get balances
-	balances, err := src.Exec_cmd(config.HarmonyPath + "./wallet.sh balances")
-	if err!=nil {
-		balances = "Error collecting data"
-	}
+	
 	
 	wrapped, err := text.New(text.WrapAtRunes())
 	if err != nil {
 		panic(err)
 	}
 
-	if err := wrapped.Write(appVersion + "", text.WriteCellOpts(cell.FgColor(cell.ColorGreen))); err != nil {
-		panic(err)
+	data.AppVersion, err = src.Exec_cmd(config.HarmonyPath + "./harmony -version")
```

Comment:
will it work without LD_LIBRARY_PATH=. env setting?

Reply:
yes, src/shell.go/Exec_cmd function is handling it